sandbox/win/src/module_file_name_interception.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_SANDBOX_WIN_SRC_MODULE_FILE_NAME_INTERCEPTION_H_
+#define BRAVE_SANDBOX_WIN_SRC_MODULE_FILE_NAME_INTERCEPTION_H_
+
+#include <Windows.h>
+
+#include <Psapi.h>
+
+#include "sandbox/win/src/sandbox_types.h"
+
+#if (PSAPI_VERSION > 1)
+#define TargetGetModuleFileNameExA TargetK32GetModuleFileNameExA
+#define TargetGetModuleFileNameExW TargetK32GetModuleFileNameExW
+#endif
+
+namespace sandbox {
+
+using GetModuleFileNameAFunction = decltype(&::GetModuleFileNameA);
+using GetModuleFileNameWFunction = decltype(&::GetModuleFileNameW);
+using GetModuleFileNameExAFunction = decltype(&::GetModuleFileNameExA);
+using GetModuleFileNameExWFunction = decltype(&::GetModuleFileNameExW);
```

Comment:
there's also `GetModuleBaseName`, but MSDN recommends `GetModuleFileName` to get the current process name.